ui/components/multichain/import-nfts-modal/import-nfts-modal.test.js

```diff
@@ -168,41 +185,112 @@ describe('ImportNftsModal', () => {
     fireEvent.click(addNftClose);
   });
 
-  it('should route to default route when cancel button is clicked', () => {
+  it('should set error message when address invalid', () => {
     const onClose = jest.fn();
-    const { getByText } = renderWithProvider(
+    const { getByText, getByPlaceholderText } = renderWithProvider(
       <ImportNftsModal onClose={onClose} />,
       store,
     );
 
-    const cancelButton = getByText('Cancel');
-    fireEvent.click(cancelButton);
+    const addressInput = getByPlaceholderText('0x...');
+    fireEvent.change(addressInput, {
+      target: { value: INVALID_ADDRESS },
+    });
 
-    expect(useHistory().push).toHaveBeenCalledWith(DEFAULT_ROUTE);
+    const errorMessage = getByText('Invalid address');
+    expect(errorMessage).toBeInTheDocument();
   });
 
-  it('should route to default route when close button is clicked', () => {
+  it('selectedNetworkClientId is for the network the NFT is on, not the globally selected network of the wallet', async () => {
+    const customNetwork = {
+      rpcUrl: 'https://alt-rpc.rpc.com',
+      chainId: '0x123',
+      ticker: 'ETH',
+      nickname: 'Custom Network',
+      id: 'custom-network-client-id',
+      blockExplorerUrl: undefined,
+    };
+
+    store = configureMockStore([thunk])({
+      ...mockState,
+      metamask: {
+        ...mockState.metamask,
+        ...mockNetworkState(
+          { chainId: CHAIN_IDS.MAINNET },
+          { chainId: CHAIN_IDS.GOERLI },
+          customNetwork,
+        ),
+      },
+    });
+
     const onClose = jest.fn();
-    renderWithProvider(<ImportNftsModal onClose={onClose} />, store);
+    const { getByText, getByPlaceholderText, getByTestId, debug } =
+      renderWithProvider(<ImportNftsModal onClose={onClose} />, store);
 
-    fireEvent.click(document.querySelector('button[aria-label="Close"]'));
+    // Click network selector button
+    const networkSelectorButton = getByTestId(
+      'test-import-tokens-drop-down-custom-import',
+    );
+    fireEvent.click(networkSelectorButton);
+
+    // Select custom network
+    const customNetworkItem = getByText('Custom Network');
+    fireEvent.click(customNetworkItem);
+
+    debug();
```

Comment:
Can we remove the debug statement?

Reply:
Welp, thank you for catching. Will remove.